Update browser support

tysongach opened the pull request:
- According to http://caniuse.com/usage-table, global usage of IE 9 and 10 are
both around 0.6%.

tute commented:
We can drop 10 already? That is great! Can you add links with information to the commit message, please? Thank you!

brendastorer commented:
:tada: :tada: :tada:

tysongach commented:
@tute Updated, and provided better messaging.

I also renamed this PR and added another commit to drop iOS 7, which has little usage these days, as well.

cc: @brendastorer

tute commented:
:+1: Thank you!
